forge/csrc/forge_passes.cpp

```diff
@@ -157,7 +157,8 @@ void run_optimization_graph_passes(graphlib::Graph *graph)
 
     passes::hoist_transforms_to_inputs(graph);
     passes::erase_consecutive_reshape(graph, true);
-    passes::lower_reinterpret_shape(graph);
+    // skip lower_reinterpret_shape in optimization passes to avoid changing expected shapes on inputs
```

Comment:
Do you see a value in fully removing this code section? If we need a reference in future, we can always check PyBuda repo.

Reply:
+1 just remove it 😄

i would also remove that pass entirely.

Reply:
Hmmm, I wanted to leave some clue why we are skipping this for now. Looking back to PyBuda doesn't sound amazing to me. Do we have some third option?

Reply:
> Do we have some third option?

To remove it and never look back? 😂